CMakeLists.txt

```diff
@@ -538,7 +538,7 @@ else()
 endif()
 
 if (SWIFT_BUILD_RUNTIME_WITH_HOST_COMPILER)
-  if (NOT CMAKE_Swift_COMPILER AND NOT SWIFT_PREBUILT_SWIFT)
+  if ("${CMAKE_Swift_COMPILER}" STREQUAL "CMAKE_Swift_COMPILER-NOTFOUND" AND NOT SWIFT_PREBUILT_SWIFT)
```

Comment:
In my testing and debugging, it needs to be: `"${CMAKE_Swift_COMPILER}" STREQUAL "NOTFOUND"`

Reply:
Ah, okay.